README.md

```diff
@@ -0,0 +1,8 @@
+# kudet
+
+kudet - pronounced cadet - stands for (ku)rtosis (de)veloper (t)ools. It is a cli that houses tools essential the workflow of a Kurtosis developer.
+
+### Supported Commands
+`kudet release`
```

Comment:
You can make this dynamic by having the user pass in the `-h` command when running the CLI

Reply:
gotcha, should I remove this from README then? I was the thinking eventually the basic cli api should be documented in the README.